playbooks/ovs_2_ml2_cleanup_remnants.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+# a cleanup script to run after a host has been migrated from OVS to ml2 / linux bridge
+
+- hosts: all
+  serial: 10
```

Comment:
Somewhere we had serial=1 defined as the default, so this was to override that. Probably not needed any more.